worker/src/RTC/StunPacket.cpp

```diff
@@ -209,6 +209,22 @@ namespace RTC
 					break;
 				}
 
+				case Attribute::NOMINATION:
+				{
+					// Ensure attribute length is 4 bytes.
+					if (attrLength != 4)
+					{
+						MS_WARN_TAG(ice, "attribute NOMINATION must be 4 bytes length, packet discarded");
+
+						delete packet;
+						return nullptr;
+					}
+
+					packet->SetNomination(Utils::Byte::Get4Bytes(attrValuePos, 0));
```

Comment:
Before this, I'd add a packet->SetHasNomination(true) which sets a new Packet.hasNomination Boolean  to true. This way, instead of checking nomination >= we can check (nomination > this.remoteNomination || !this.hasNomination). What do you think?

Reply:
Here we set `nomination = 1` when the packet has USE-CANDIDATE attr. And chorme will never send STUN packet with NORMINATION attr and value = 0 or value equals previous tuple.
if we want to make it more readable, we need `hasCandidate`, `hasNomination`, `normination` as the args.
So `if(nomination == 0)` means `if(!hasUseCandidate && !hasNomination)`
`if(nomination >= thisremoteNomination)` means `if(hasUseCandidate || (hasNomination && normination > this.remoteNomination))`
Which one do you perferred?

Reply:
I'm handling this in this branch directly, will push soon.